src/prometheus_alert_rules/reboot.rules

```diff
@@ -0,0 +1,14 @@
+groups:
+- name: HostReboot
+  rules:
+  - alert: HostReboot
+    expr: changes(node_boot_time_seconds[5m]) > 0 and node_boot_time_seconds != (node_boot_time_seconds offset 1m)
```

Comment:
a few questions:
- will this detect a reboot that happened only in the past 5 minutes?
- for how long will this keep alerting after a reboot?

Reply:
Since we are using `for: 0m` it will alert immediately and only once (if `evaluation_interval` is 1m), because `node_boot_time_seconds != (node_boot_time_seconds offset 1m)`. If we remove `node_boot_time_seconds != (node_boot_time_seconds offset 1m)` it will fire 4-5 times, depends how long machine will be out.

If machine is down for more than 5 minute this will no fire anything.